src/containers/my-quizzes/QuizzesContainer.tsx

```diff
@@ -63,25 +69,18 @@ const TestsContainer = () => {
     }
   ]
 
-  return (
-    <Box>
-      {loading ? (
-        <Loader pageLoad size={50} />
-      ) : (
-        <EnhancedTable
-          columns={columnsToShow}
-          data={{ items: response.items }}
-          emptyTableKey='myResourcesPage.quizzes.emptyQuizzes'
-          rowActions={rowActions}
-          sort={{
-            sort: { order: SortEnum.Desc, orderBy: 'updatedAt' },
-            onRequestSort: () => null
-          }}
-          sx={styles.table}
-        />
-      )}
-    </Box>
+  const tableAttachments = (
+    <EnhancedTable
+      columns={columnsToShow}
+      data={{ items: response.items }}
+      emptyTableKey='myResourcesPage.quizzes.emptyQuizzes'
+      rowActions={rowActions}
+      sort={{ sort, onRequestSort }}
```

Comment:
you can pass sortOptions here: 
sort={sortOptions}

Reply:
> you can pass sortOptions here: sort={sortOptions}

fixed that and removed unnecessary variable